### ISSUE TITLE
Way to disable keyboard shortcuts

### ISSUE THREAD

**Issue by younishd** [hackmdio/codimd#530]
It would be nice to have a feature to disable the keyboard shortcuts.

**Comment by younishd** [comment]
`Ctrl+T` seems to flip the two characters that are on the left and right of the cursor, but it also opens a new tab in most browsers.

The last time I checked (e.g., when I opened this issue) it kept doing both at the same time if I accidentally pressed it while in edit mode, which was very annoying so I stopped doing that and used the mouse to open a new tab instead while editing.
I just retested it now and it seems like browser shortcuts are now correctly overridden as it no longer opens a new tab when I press `Ctrl+T` (although the "Allow override browser keymap" option has no effect on that).

It would be great to have a way of disabling a shortcut like `Ctrl+T` so that the browser would just open a new tab.
